src/appleseed.cli/CMakeLists.txt

```diff
@@ -59,11 +59,11 @@ add_executable (appleseed.cli
     ${appleseed.cli_sources}
 )
 
-if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+if (UNIX AND NOT WIN32 AND NOT APPLE)
```

Comment:
As we are doing this test multiple times, wouldn't it make sense to do the test once, save the result in a cmake variable and test that instead?

something like (in the top level CMakefile)

if (UNIX AND NOT APPLE AND NOT CYGWIN)
   set (APPLESEED_IS_UNIX True)
endif ()

?